m2Final.js

```diff
@@ -0,0 +1,58 @@
+const select = document.querySelector("#movie-titles");
+const chosenTitle = document.querySelector("#selected-title");
+const chosenRelease = document.querySelector("#release-yr");
+const chosenDescription = document.querySelector("#film-description");
+const reviewList = document.querySelector("#review-list");
+const reviewSubmission = document.querySelector("#review-submit");
+
+const createOptions = async () => {
+  try {
+    const option = document.createElement("option");
+    option.textContent = "";
+    select.appendChild(option);
+    const res = await axios.get("https://ghibliapi.herokuapp.com/films/");
+    res.data.forEach((movie) => {
+      const option = document.createElement("option");
+      option.textContent = movie.title;
+      select.appendChild(option);
+    });
+  } catch (err) {
+    console.log(err);
+  }
+};
+createOptions();
+
+const movieInfo = async () => {
+  try {
+    const res = await axios.get("https://ghibliapi.herokuapp.com/films/");
+    res.data.forEach((film) => {
+      if (film.title === chosenTitle.textContent) {
+        chosenTitle.textContent = film.title;
+        chosenRelease.textContent = film.release_date;
+        chosenDescription.textContent = film.description;
+      }
+    });
```

Comment:
With the value property added to the option object we can make a call to a single film. The api call here should be to a [single film using its id](https://ghibliapi.herokuapp.com/#tag/Films%2Fpaths%2F~1films~1%7Bid%7D%2Fget).
 To get the id of the film that was chosen by the select dropdown we can utilize the `event` object ==> `e.target.value` or from the select variable `select.value`. The `value` property was used before to store the id of that movie. Once we get the data we just reassign the `innerText` for the title, release date, and description.